src/internal/storage/btree/BtreePrinter.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <fstream>
+#include <iterator>
+#include <sstream>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <internal/storage/btree/Btree.h>
+
+namespace internal::btree::util {
```

Comment:
For me this identation is very strange. I think that everything in curly braces must be with a tab. In Btree.h the namespace has the same problem.

Reply:
Do you mean that the file should like something like this:
```cpp

namespace foo {
    class Bar {
        ...
    };
}

```

Instead of 

```cpp

namespace foo {

class Bar {
    ...
};

}

```

Reply:
In my opinion this is the cleaner way to do this since at the end of the day looking and reading code which is indented 3-4 times is not very pleasant. Therefore minimizing it as much as possible without sacrificing readability is welcome.